system-dependencies.sh

```diff
@@ -311,7 +311,9 @@ function download_xcode_platforms ()
 		return
 	fi
 
+	log "Executing '$SUDO $XCODE_DEVELOPER_ROOT/usr/bin/xcodebuild -downloadAllPlatforms'"
 	$SUDO "$XCODE_DEVELOPER_ROOT/usr/bin/xcodebuild" -downloadAllPlatforms
+	log "Executing '$SUDO $XCODE_DEVELOPER_ROOT/usr/bin/xcodebuild -downloadAllPlatforms'"
```

Comment:
Maybe? 
```suggestion
	log "Done Executing '$SUDO $XCODE_DEVELOPER_ROOT/usr/bin/xcodebuild -downloadAllPlatforms'"
```

Reply:
Fixed, I forgot to save at some point :/